tilecloud_chain/internal_mapcache.py

```diff
@@ -133,9 +148,20 @@ def __init__(self, tilegeneration):
         self._cache_store = RedisStore(
             redis_config["url"], redis_config["prefix"], redis_config["expiration"]
         )
-        for i in range(10):
-            thread = GeneratorThread(i, tilegeneration, self._input_store, self._cache_store)
+        self.threads = []
+        generator = Generate(FakeOptions(), tilegeneration, server=True)
+        generator.server_init(self._input_store, self._cache_store)
+        for i in range(int(os.environ.get("SERVER_NB_THREAD", 10))):
```

Comment:
might there be a need to continue supporting single threading (i.e. not using too many CPU threads)? If yes I would probably put the default value for `SERVER_NB_THREAD` to 1 here

Reply:
Continue is a big word because before it was already multiheaded... and hard-coded to use 10 threads..., and it looks to be a big change to don't use a therad...